management.tex

```diff
@@ -1,3 +1,79 @@
+
+Data Facility representatives were present at this meeting even though all the processing was being done on the Interim Data Facility at Google \citep{2021arXiv211115030O}.
+
+\subsection{Work Management}
+
+We used Jira to track work related to the Data Preview.
+Epics and milestones were created in the \texttt{PREOPS} Jira Project.
+Story tickets were then attached to each epic but in order to properly integrate the work into existing Data Management processes, any tickets that would result in code changes in pipelines software or middleware packages were created in the Data Management Jira project.
+
+The status of the epics and how they related to the relevant milestones was monitored as part of the weekly coordination or DPLT meetings.
+
+\subsection{Change Control}
+
+Data Preview 0.2 used v23.0.x of the LSST Science Pipelines Software and that was derived from a weekly release from September 2021 (\texttt{w.2022.40}).
+We decided to group processing into distinct ``steps'' that allowed updates to the software used in later stages of processing to be worked on whilst earlier steps were executing.
+
+We continued to want to use the v23 release for all data processing and that required that we had a process to determine which patches would need to be back-ported to the release branch as needed before each step could begin.
+
+The Data Management Change Control Board (DMCCB) and DPLT delegated authority to a new Campaign Management Board that had the following membership:
+
+\begin{itemize}
+\item Yusra AlSayyad, representing the pipelines team.
+\item Leanne Guy and Colin Slater, representing the verification and validation team.
+\item Hsin-Fang Chiang, representing the execution team.
+\item Tim Jenness, representing the data processing architecture team.
+\end{itemize}
+
+The Board met weekly on Tuesday at 8:30am Pacific Time and also had a Slack channel to discuss any issues that would come up between meetings.
+Minutes for the meetings were recorded on Confluence.
+
+The process for deciding on a back-port is as follows:
+
+\begin{enumerate}
+
+\item A request is made that a ticket should be applied to the release branch by applying a \texttt{backport-v23} tag to the Jira ticket.
+\item The board would then discuss the relative merits of the back-port and if approved a \texttt{backport-approved} label would be added.
+\item The work on the back-port would then be scheduled by the relevant T/CAM following instructions in the developer guide.\footnote{\url{https://developer.lsst.io/work/backports.html}}
+\item Once the code is on the \texttt{v23.0.x} branch a \texttt{backport-done} label would be applied.
+
+\end{enumerate}
+
+A Jira query was constructed to find all the tickets and track their porting status.
+There were 64 tickets with back-port requests and 61 of those were approved and applied to the release branch.
```

Comment:
Turns out that when a ticket was rejected we removed the label so it's now not easy to find out how many we rejected. @ctslater pointed this out to me and I had forgotten. The three described here were ones that we couldn't decide whether to do or not...